src/activation/activation.rs

```diff
@@ -8,11 +8,13 @@ pub enum Activation {
     ReLu,
     Tanh,
     Softmax,
+    Sigmoid,
 }
 
 impl Activation {
     pub fn activate<D: Dimension>(&self, inputs: &Tensor<D>) -> Tensor<D> {
         match self {
+            Self::Sigmoid => inputs.mapv(|input| self.sigmoid(input)),
```

Comment:
Very nitpicky, but can you place this line below the `Softmax` case to follow the same order as the Enum?